examples/cifar10/cifar10-sim/job_configs/cifar10_autofedrl/cifar10_autofedrl/config/config_fed_server.json

```diff
@@ -0,0 +1,148 @@
+        },
+        {
+            "id": "persistor",
+            "name": "PTFileModelPersistor",
+            "args": {
+                "model": "model"
+            }
+        },
+        {
+            "id": "shareable_generator",
+            "name": "PTFedOptModelShareableGenerator",
+            "args": {
+                "device": "cpu",
+                "source_model": "model",
+                "optimizer_args": {
+                    "path": "torch.optim.SGD",
+                    "args": {
+                        "lr": 1.0,
+                        "momentum": 0.0
+                    }
+                }
+            }
+        },
+        {
+            "id": "search_space",
+            "name": "PTAutoFedRLSearchSpace",
+            "args": {
+                "search_type": "cs",
+                "device": "cpu",
+                "optimizer_args": {
+                    "path": "torch.optim.Adam",
+                    "args": {
+                        "lr": 0.01,
+                        "betas": [
+                            0.7,
+                            0.7
+                        ]
+                    }
+                },
+                "n_clients": "{min_clients}",
+                "search_lr": true,
+                "lr_range": [
+                    0.0005,
+                    0.05
+                ],
+                "search_ne": true,
+                "ne_range": [
+                    2,
+                    40
+                ],
+                "search_aw": true,
+                "aw_range": [
+                    0.1,
+                    1.0
+                ],
+                "search_slr": true,
+                "slr_range": [
+                    0.5,
+                    1.5
+                ],
+                "initial_precision": 85.0
+            }
+        },
+        {
+            "id": "aggregator",
+            "name": "InTimeAccumulateWeightedAggregator",
+            "args": {}
+        },
+        {
+            "id": "aggregator_search_val",
+            "name": "InTimeAccumulateWeightedAggregator",
+            "args": {
+                "expected_data_kind": "METRICS",
+                "aggregation_weights": {
+                    "site-1": 1.0,
+                    "site-2": 1.0,
+                    "site-3": 1.0,
+                    "site-4": 1.0,
+                    "site-5": 1.0,
+                    "site-6": 1.0,
+                    "site-7": 1.0,
+                    "site-8": 1.0
+                }
+            }
+        },
+        {
+            "id": "model_selector",
+            "name": "IntimeModelSelector",
+            "args": {}
+        },
+        {
+            "id": "model_locator",
+            "name": "PTFileModelLocator",
+            "args": {
+                "pt_persistor_id": "persistor"
+            }
+        },
+        {
+            "id": "json_generator",
+            "name": "ValidationJsonGenerator",
+            "args": {}
+        }
```

Comment:
Is this example using these components?
(model_selector, model_locator, json_generator)

If not maybe we don't include them in config here?